charts/redpanda/ci/13-nodeport-tls-values.yaml

```diff
@@ -0,0 +1,40 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+---
+external:
+  enabled: true
+  type: NodePort
+    # If specified, then it will be appended to the `external.addresses` values as each broker's advertised address
+    # domain: local
+    # Optional list of addresses that the Redpanda brokers advertise.
+    # Provide one entry for each broker in order of StatefulSet replicas.
+    # The number of brokers is defined in statefulset.replicas.
+    # The values can be IP addresses or DNS names.
+    # If external.domain is set, the domain is appended to these values.
+    # addresses:
+    # - redpanda-0
+    # - redpanda-1
+    # - redpanda-2
+    #
+    # annotations:
+    # For example:
+  # cloud.google.com/load-balancer-type: "Internal"
+  # service.beta.kubernetes.io/aws-load-balancer-type: nlb
+  # If you enable externalDns, each LoadBalancer service instance
+  # will be annotated with external-dns hostname
+  # machine external.addresses + external.domain
+  # externalDns:
+  #   enabled: true
```

Comment:
Why this commented block is added?

Reply:
This is a work in progress, I was adding code and left it since I had to switch context to help with the nightlies, I will come back to this soon